app/components/home/mailing_list_component.html.erb

```diff
@@ -0,0 +1,30 @@
+<section class="row">
+  <div class="home-mailing-list">
+    <div class="col col-full-content">
+      <div>
+        <h2><i class="icon icon-arrow-black icon--on-yellow"></i> Sign up for our emails</h2>
+        <p>
+          Everything you need to know to start a career in teaching sent straight into your inbox.
+          Tailored to your own situation, you get all the latest information as well as advice and support.
```

Comment:
`you'll get` or `you will get`? but happy for you to disagree and leave as is @philspray @EllieNodder

Reply:
probs `you'll get`?